ViennaAdvantage/Areas/VA012/Models/BankStatementModel.cs

```diff
@@ -4095,6 +4215,9 @@ public string CreatePaymentFromSchedule(Ctx ctx, List<StatementProp> _formData)
                                     return !string.IsNullOrEmpty(error) ? error : "VA012_PaymentNotSaved";
                                 }
                             }
+                            //fetch the Updated data from new Instance 
+                            _pay = new MPayment(ctx, _pay.GetC_Payment_ID(), null);
```

Comment:
implement Complete Action call by worker, not to call direct

Reply:
Done.